clearnode/docs/API.md

```diff
@@ -714,6 +714,7 @@ Example:
 - User wants to deposit 75 usdc on Celo. User calls `resize_channel`, with `allocate_amount=0` and `resize_amount=75`.
 - Now user's unified balance is 100 usdc (20 on Polygon and 80 on Celo).
 - Now user wants wo withdraw all 100 usdc on Polygon. To withdraw, user must allocate 80 on this specific channel (`allocate_amount=80`), and resize it (`resize_amount=-100`). Also it is recommended to deallocate the channel on Celo (optional, but we may make this required in the future).
+- Note: all amounts are shown as examples. In your requests, you must specify these amounts in wei.
```

Comment:
I would not use the term `wei` not to create a connection with Ether.
What do you think about the following parlance: "... must specify these amounts as not formatted with decimals"?